src/store/recordAction.ts

```diff
@@ -0,0 +1,13 @@
+import * as FullStory from '@fullstory/browser';
+import { Middleware } from 'redux';
+
+const recordAction: Middleware = () => next => action => {
```

Comment:
Thinking about configuration for this - there might be a category of actions where people might find it valuable to see a stringified JSON payload of the entire action, perhaps we could follow some of the same patterns as other logging middleware and provide a function that accepts the action as an argument, and returns true/false if it should be logged in full?

Reply:
Yeah, I can see that as a v2 - I'm hoping that we can provide value without having to wade into privacy waters and then evaluate recording payloads later.